### DISCUSSION TITLE
Turn off ".returning() is not supported by mysql and will not have any effect."

### DISCUSSION THREAD

**Discussion by facku** [adonisjs/core#1280] [Help] [answered]
Hi. I'm using Adonis v4.1 with mysql (MariaDb). How i can disable/turn off the following message  in console when I run migrations and seeders:

`.returning() is not supported by mysql and will not have any effect.´`

Thanks a lot.

**Comment by benknight** [comment]
I'm also seeing this issue. It appears to be happening in `node_modules/@adonis/lucid/src/Model/index.js` line 644 (@adonis/lucid@6.2.0). The insert query is always chains a `.returning` call a knex insert query which knex triggers a warning for if you're using mysql. Unfortunately just have to ignore it but it's polluting my console with hundreds of warnings which is quite frustrating.

**Reply by thetutlage** [reply]
Mind sharing a PR for the fix?

**Reply by benknight** [reply]
Sure I'll give it a swing

**Reply by benknight** [reply]
I tried and failed to fix this.

https://github.com/adonisjs/lucid/issues/571#issuecomment-665431681

**Reply by woodgates** [reply]
I don't know why but I started receiving this message on almost every query. This happened this month after doing a npm install  . Very annoying.

**Comment by Agilulfo1820** [comment]
Same problem. Very annoying and I'm also filling CloudWatch logs basically only with this warning.

**Comment by benknight** [comment, marked as answer]
Update - this has been fixed as of v6.2.1

**Reply by benknight** [reply]
@facku @Agilulfo1820 @woodgates fyi ^

**Reply by Agilulfo1820** [reply]
~~Do you refer to lucid version 6.2.1? Because I downloaded that version of lucid but still have this issue :(~~

EDIT: Everything works perfectly! Thank you guys!

**Reply by woodgates** [reply]
@benknight thanks for heads up! I updated lucid and for the first minutes I don't get the error

**Reply by mritg** [reply]
@benknight For me still i am getting warnings about either 6.2.1 or, the latest
